stdlib/public/Platform/CMakeLists.txt

```diff
@@ -56,7 +57,11 @@ foreach(sdk ${SWIFT_SDKS})
       set(GLIBC_SYSROOT_RELATIVE_INCLUDE_PATH "/system/develop/headers/")
     else()
       # Determine the location of glibc headers based on the target.
-      set(GLIBC_SYSROOT_RELATIVE_INCLUDE_PATH "/usr/include")
+      if(${sdk} STREQUAL "FUCHSIA")
```

Comment:
Nitpick: `elseif`?

Reply:
Yeah sorry. I wrote this before the Haiku port landed and just wrapped what it was doing in my version in the else in the merge. Will rethink this. 

Part of it is borked anyways because the module is baking in absolute paths on the machine that compiled it (which is fine when you have absolute paths that reference well known paths like /usr/include but not fine when you have a sysroot for another machine in some directory). We need to get VFS mappable module importer working to support SDKs that are relocatable with glibc.modulemap properly. Different problem to solve though after this to get cross compiling working properly. For us we have a libc.modulemap in Fuchsia that people will use instead of the Glibc module map (we only use glibc module map for tests right now). Would like to get things working cleaner though everywhere.